web/src/components/molecules/Member/MemberAddModal/index.tsx

```diff
@@ -66,28 +68,18 @@ const MemberAddModal: React.FC<Props> = ({
     [changeSearchedUserList],
   );
 
-  const handleSubmit = useCallback(() => {
-    form
-      .validateFields()
-      .then(() => {
-        if (searchedUserList.length > 0) {
-          onSubmit(
-            searchedUserList.map(user => {
-              return {
-                userId: user.id,
-                role: "READER",
-              };
-            }),
-          );
-        }
-        changeSearchedUser(undefined);
-        changeSearchedUserList([]);
-        onClose();
-        form.resetFields();
-      })
-      .catch(info => {
-        console.log("Validate Failed:", info);
-      });
+  const handleSubmit = useCallback(async () => {
+    if (searchedUserList.length === 0) return;
+    await onSubmit(
+      searchedUserList.map(user => ({
+        userId: user.id,
+        role: "READER",
+      })),
+    );
+    changeSearchedUser(undefined);
+    changeSearchedUserList([]);
+    onClose();
+    form.resetFields();
```

Comment:
try, catch... statement should be added here

Reply:
There is no process to do within the scope of catch, but is it necessary to use try-catch?
Or is there error handling that should be done?

Reply:
I think for now it is okay to do something like this:
```
try {
  // some code
} catch(error) {
  console.error(error);
}
```
maybe later we can work on adding/displaying error messages

Reply:
Also these should go in the `finally` block:
```
    changeSearchedUser(undefined);
    changeSearchedUserList([]);
    onClose();
    form.resetFields();
```

Reply:
> maybe later we can work on adding/displaying error messages

I got it. Is this code better?
```
await onSubmit(
      searchedUserList.map(user => ({
        userId: user.id,
        role: "READER",
      })),
    ).catch(error => {
      console.error(error);
    });
```

> Also these should go in the finally block:

I don't think we can go them in the `finally` block because we want to process them only when onSubmit succeeds.

Reply:
Yep, you are right. No need to move them to the finally block.

Reply:
> await onSubmit(
>       searchedUserList.map(user => ({
>         userId: user.id,
>         role: "READER",
>       })),
>     ).catch(error => {
>       console.error(error);
>     });

I think since you are using async... await, the try... catch statement is more appropriate. right?

Reply:
It would be better to use try...catch statement. Thank you for your advice. 🙇‍♂️

Reply:
Thank you 🙇🏻